src/applications/CMakeLists.txt

```diff
@@ -1,12 +1,12 @@
-cmake_minimum_required(VERSION 3.4)
+cmake_minimum_required(VERSION 3.14)
```

Comment:
same here

Reply:
Boost::stacktrace has been added in boost 1.65 [ref](https://www.boost.org/users/history/version_1_65_0.html) and the FindBoost (versioned in CMake...) has been added in 3.13 [ref](https://gitlab.kitware.com/cmake/cmake/merge_requests/2541). So I can eventually downgrade the requirement to 3.13.